jekyll/_cci2/local-cli.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: classic-docs
+title: "The CircleCI Local Command Line Interface (CLI)"
+short-title: "Using the CircleCI Local CLI"
+description: "How to run local jobs with the CLI"
+categories: [troubleshooting]
+order: 10
+alias: /docs/2.0/local-jobs/
```

Comment:
local-jobs -> local-cli

Reply:
Actually I believe I was suppose to remove that line completely. The redirect is done using the YAML Front Matter in the original file `local-jobs`.